specifyweb/frontend/js_src/lib/components/WorkBench/hooks.ts

```diff
@@ -568,6 +568,16 @@ export function getHotHooks(wbView: WbView) {
           10
         );
     },
+    afterSelection: () => {
+      if (wbView.hot === undefined) return;
+      console.log(wbView.hot.getSelected());
+      const selection = wbView.hot?.getSelected() ?? [];
+      const newSelection = f
+        .unique(selection.map((row) => JSON.stringify(row)))
+        .map((row) => JSON.parse(row));
+      if (newSelection.length !== selection.length)
+        wbView.hot?.selectCells(newSelection);
```

Comment:
in response to:

> TODO: instead of adding newSelection to the array it needs to replace it?

You need to double check if the call to selectCells on line 579 replaces all selected cells with the newSelection, OR adds newSelection to already selected cells.

I think it would replace the existing selection, but you have to check.
If it adds rather than replaces, then you would have to look though Handsontable's documentation for a way to deselect cells